backend/src/controllers/account.controller.ts

```diff
@@ -0,0 +1,78 @@
+import { Router, RouterOptions } from 'express'
+import IAccountService from '../services/account.service'
+import { validateEmail, validatePassword } from "../models/validators";
+import { AccountDataDTO } from "../dto/account.dto";
+import { cleanUpSpaces } from "../utils/strings";
+
+
+function toAccountDTO(email: string, name?: string): AccountDataDTO {
+    email = email ? cleanUpSpaces(email.toLowerCase()) : '';
+    name = name ? cleanUpSpaces(name.toLowerCase()) : '';
+    return { name: name, email: email }
+}
+
+export default function accountController(accountService: IAccountService, routerOptions?: RouterOptions) {
+
+    const router: Router = Router(routerOptions)
+
+    router.post("/register", async (req, res) => {
+        let accountName = req.body.accountName
+        let email = req.body.email
+        let password = req.body.password
+        let account: AccountDataDTO
+
+        try {
+            // TODO: create validationUserName(accountName)
```

Comment:
are this comments needed? do you want to create a ticket for doing this? 

Reply:
No, it's already done. Thank you.